completion/api/v1/views.py

```diff
@@ -13,7 +13,9 @@
 from rest_framework.response import Response
 from rest_framework import permissions
 from rest_framework import status
+from rest_framework.authentication import SessionAuthentication
```

Comment:
You no longer need `SessionAuthentication` since `SessionAuthenticationAllowInactiveUser` is already more liberal than that.  You can remove this import and other references to it in this file.